src/main/java/org/folio/marccat/resources/BaseResource.java

```diff
@@ -34,5 +48,68 @@
     return new ResponseEntity<>(errors, HttpStatus.NOT_FOUND);
   }
 
+  public void getEmptyRecord(final boolean isBib, final Record record, final RecordTemplate template, final String lang,
```

Comment:
You can move the method to the RecordUtils class

Reply:
I have add another class